api.planx.uk/webhooks/sanitiseApplicationData/operations.ts

```diff
@@ -181,7 +181,7 @@ export const sanitiseReconciliationRequests: Operation = async () => {
       } 
     }
   `;
-  const { update_reconciliation_requests: {
+  const { delete_reconciliation_requests: {
```

Comment:
Here's the line I missed previously - this did not correspond with the query on line 173